src/pages/sectionMeeting/sectionMeeting.js

```diff
@@ -102,7 +102,6 @@ router.post(
       try {
         await dbDriver.asyncQuery(sql.insert_swipes, params)
       } catch (e) {
-        // 23505 == unique_violation, aka primary key violation here
         if (e.code && e.code === UNIQUE_VIOLATION) {
           // TODO: some nice UI which says it's a duplicate swipe
           ERR(new Error('Duplicate swipe!'))
```

Comment:
Shouldn't `ERR` be taking `next` as an argument? :confused:

Reply:
I'm not sure, but this works :x

Reply:
We can PR it into a flash now :)